test/unit/lifecycle/properties.js

```diff
@@ -104,6 +104,39 @@ describe('lifecycle/property', () => {
         expect(elem.getAttribute('test-name')).to.equal('something else');
       });
 
+      it('setting the attribute updates the property value', done => {
```

Comment:
maybe move this test to others like this? somewhere here: https://github.com/skatejs/skatejs/blob/master/test/unit/lifecycle/properties.js#L320